src/kOS/Screen/TermWindow.cs

```diff
@@ -256,6 +258,15 @@ private void GetFontIfChanged()
 
                 terminalLetterSkin.label.font = font;
                 terminalLetterSkin.label.fontSize = fontSize;
+                if (recalcWidth)
+                {
+                    CharacterInfo chInfo;
+                    terminalLetterSkin.label.font.RequestCharactersInTexture("X"); // Make sure the char in the font is lazy-loaded by Unity.
+                    terminalLetterSkin.label.font.GetCharacterInfo('X', out chInfo);
+                    shared.Screen.CharacterPixelWidth = chInfo.advance;
+
+                    NotifyOfScreenResize(shared.Screen);
+                }
```

Comment:
The work above is work that used to be done inside TerminalGUI(), partway through drawing things.  Now it's happening  here, which occurs at the very start of OnGUI() rather than halfway through it.